lib/Onyx.js

```diff
@@ -364,7 +364,7 @@ function keysChanged(collectionKey, partialCollection) {
                 // returned by the selector.
                 if (subscriber.selector) {
                     subscriber.withOnyxInstance.setState((prevState) => {
-                        const previousData = reduceCollectionWithSelector(prevState[subscriber.statePropertyName], subscriber.selector, subscriber.withOnyxInstance.state);
+                        const previousData = prevState[subscriber.statePropertyName];
```

Comment:
Ah hmm so we were "selecting" the already selected data in state? And it would always be different. Makes sense. Great catch.

Reply:
Yes, exactly

Reply:
Awesome! You are a gem! 💎